packages/token-providers/package.json

```diff
@@ -6,8 +6,8 @@
   "module": "./dist-es/index.js",
   "sideEffects": false,
   "scripts": {
-    "build:client-dist": "node ./scripts/esbuild",
-    "build": "yarn build:client-dist && concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
```

Comment:
this is the main difference. Instead of trying to build a bundle during the build of this package, the bundle is static. 

While it may fall out of sync with the SSO OIDC client, this is necessary anyway to maintain a minimal version of the client that does not need the credential provider chain.